rundeckapp/build.gradle

```
@@ -176,6 +176,18 @@ dependencies {
         exclude(group:"org.grails.profiles",module:"web")
     }
 
+    // OpenApi Dependencies
+
+    annotationProcessor "io.micronaut.openapi:micronaut-openapi:4.0.0"
+    implementation "io.swagger.core.v3:swagger-annotations:2.1.1"
+    compileOnly "io.micronaut.openapi:micronaut-openapi:4.0.0"
+    implementation 'io.micronaut:micronaut-http-server:${micronautVersion}'
+    implementation 'io.micronaut:micronaut-inject:${micronautVersion}'
+    implementation 'io.micronaut:micronaut-inject-java:${micronautVersion}'
+    implementation 'io.micronaut:micronaut-inject-groovy:${micronautVersion}'
+    implementation 'io.micronaut:micronaut-core:${micronautVersion}'
```

single quotes won't expand the version data
```suggestion
    implementation "io.micronaut:micronaut-http-server:${micronautVersion}"
    implementation "io.micronaut:micronaut-inject:${micronautVersion}"
    implementation "io.micronaut:micronaut-inject-java:${micronautVersion}"
    implementation "io.micronaut:micronaut-inject-groovy:${micronautVersion}"
    implementation "io.micronaut:micronaut-core:${micronautVersion}"
```